pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -272,79 +272,43 @@ def newpkgcb(pkgId, name, arch):
         await out_q.put(None)
 
 
-class ErrataRelatedModelSaver(Stage):
+class RPMContentUnitSaver(ContentUnitSaver):
```

Comment:
If this is only for Erratum (my impression based on the comments/docstrings), we should probably name it something more specific.